README.md

```diff
@@ -33,15 +33,77 @@ e.g: (for script named `script.py`)
 1. If `log_file_prefix=prefix` --> log file will be stored at `./prefix_wesci_log`
 1. If `log_file_prefix=/log/prefix` --> log file will be stored at `/log/prefix_wesci_log`
 
-### Adding Input/Output Parameters and Files
+### Logging Params
+**Note:** We don't support all variable types yet. When an unsupported type is passed as input/output param, a warning will be issued when the code is executed, and the param will be ignored.
```

Comment:
Maybe we should have a list/table of the supported types we do cover along with a detailed description of what exactly we capture (And send) to the cloud...

Reply:
I thought about it - but we don't even know what we support yet.
It's too early and we have too few users for that.
Keeping with the MVP spirit - I think this is enough

Reply:
Yah, Probably right...